hibike/devices/YogiBear/motor.cpp

```diff
@@ -38,13 +39,15 @@ float readCurrent() {
 //takes a value from -1 to 1 inclusive and writes to the motor and sets the INA and INB pins for direction
 void drive(float target) {
   
-  if (target < 0) { 
+  if (target < -deadBand) { 
     digitalWrite(INA, LOW);
     digitalWrite(INB, HIGH);
     target = target * -1;
-  } else if (target > 0) {
+  } else if (target > deadBand) {
     digitalWrite(INA, HIGH);
     digitalWrite(INB, LOW);
+  } else {
+    target = 0;
```

Comment:
no, motor disable resets all values and states and should only be used when hibike issues a disable command. In this case, we want the robot to continue its operation as it was before stopping so we will just set the PWM to 0